engine/src/physics/solver/penetration_constraint/plugin.cpp

```diff
@@ -120,7 +121,7 @@ static void solvePenetrationConstraint(
             }
             else if (useBias)
             {
-                bias = glm::max(constraint.biasCoefficient * separation, -4.0F);
+                bias = glm::max(constraint.biasCoefficient * separation, physicsConstants.biasMax);
```

Comment:
I have no  knowledge about this code, but if we're taking the max of two quantities, shouldn't biasMax be called biasMin?

Reply:
It might be wrong, I don't really have knowledge about some aspects of the code or its magic numbers context.
I based it off of its "position in the code", I read "bias", I read "max", so I called it "biasMax", but that might be wrong yes.
Tiago might be able to clarify this as I actually need someone who knows the code to check if the names are all right.

Reply:
Yeah, it's a bit of a weird case because it is the maximum bias, but the bias happens to be negative so it's treated as the minimum. I think the best would be to maybe keep biasMax but put a comment on the resource saying the bias is negative and so it is the maximum negative value it can have, just so there is no confusion in the future.